src/parser/skewMatrix.ts

```diff
@@ -1,20 +1,47 @@
 import { degreesToRadians } from '../util/misc/radiansDegreesConversion';
-import { TMat2D } from '../typedefs';
+import { TDegree, TMat2D } from '../typedefs';
 
 /**
  * A matrix in the form
- * [0 x 0]
- * [y 0 0]
+ * [1 x 0]
+ * [0 1 0]
  * [0 0 1]
+ *
+ * or
+ *
+ * [1 0 0]
+ * [y 1 0]
+ * [0 0 1]
+ *
  * For more info, see
  * @link https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/transform#skewx
  */
-type TMatSkew = [xy: number];
+const fromAngleToSkew = (angle: TDegree) => Math.tan(degreesToRadians(angle));
```

Comment:
@asturur this is a valid util, should belong to utils IMO
I will move it in #8893